src/main/java/cubicchunks/lighting/LightingManager.java

```diff
@@ -214,16 +209,66 @@ void markBlockColumnForUpdate(int localX, int localZ) {
         }
 
         public void tick() {
+            LightUpdateTracker tracker = cube.getCubicWorld().getLightingManager().getTracker();
+            for(EnumFacing dir:EnumFacing.values()){
+                if(cube.edgeNeedSkyLightUpdate[dir.ordinal()]){
```

Comment:
`if(` -> `if (`, `for(` -> `for (`, `EnumFacing dir:EnumFacing.values()` -> `EnumFacing dir : EnumFacing.values()`